src/guide/1/essentials/user-authentication/index.md

```diff
@@ -19,10 +19,10 @@ Additionally, you can set:
 * [User credentials]({{ site_base_path }}guide/1/essentials/user-authentication/#user-credentials): If no credentials are provided, then the user cannot [login]({{ site_base_path }}api/1/controller-auth/login/)
 * Any number of properties that you want to store in your user object, such as a lastname or a list of hobbies. These properties are stored at the user level and are not linked to any particular authentication strategy.
 
-Let's create a user with username `jondoe` and password `letmein` through the API:
+Let's create a user with username `admin` and password `test` through the API:
```

Comment:
I don't get why you changed these example values.

Firstly, I think that `johndoe` is more appropriate than`admin`, since we're describing how to create a user with the `default` profile, and not with the `admin` one.

Also, this is a nitpicking, but I kind of like the `letmein` password, it's funnier and actually better suited than `test`.

Reply:
I changed it because I thought there was a mistake : admin and jondoe was mentioned as username, letmein and test was mentioned as password, for the same example.

Reply:
Ahh, I see. You're right to homogenize these values, but I think you chose the wrong ones :slightly_smiling_face:

Reply:
Noted :)